integ/aws/compute/apigw_test.go

```diff
@@ -0,0 +1,152 @@
+package test
+
+import (
+	"fmt"
+	"net/url"
+	"testing"
+	"time"
+
+	http_helper "github.com/gruntwork-io/terratest/modules/http-helper"
+	"github.com/stretchr/testify/require"
+
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
+	util "github.com/terraconstructs/base/integ/aws"
+)
+
+// Test the apigw.token-authorizer app
+func TestApigwTokenAuthorizer(t *testing.T) {
+	runComputeIntegrationTest(t, "apigw.token-authorizer", region, func(t *testing.T, tfWorkingDir, awsRegion string) {
+
+		// TODO: Find out why API Gateways fail until they are redeployed??
+		// force re-deployment of the API Gateway to ensure the latest changes are applied
+		util.ReplaceTerraformResource(t, tfWorkingDir, "aws_api_gateway_deployment", "")
```

Comment:
could be related to https://github.com/TerraConstructs/base/issues/51 ?

Strange there is no error during the apply (but I suspect the deployment to stage is triggered when the RestApi config isn't fully consistent yet.. which is why I need to trigger a re-deploy)